### PR TITLE
created alerts and monitoring readmes

### PR DESCRIPTION
**Pull request by sudoku-lord**

### CODE REVIEW

File: MONITORING.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+- [Test Scripts](#test-scripts)
+- [Alerts](#alerts)
+
+## Introduction
+
+Monitoring tests exist for all of the endpoints specified by Lob OpenAPI. These tests are automatically scheduled to run every 5 minutes, against the production version of each endpoint. The following document will explain all the components which go into running monitoring tests smoothly.
+
+## Monitoring in GitHub
+
+We use GitHub Actions to run the monitoring tests. To view the monitor runs, click on the Actions tab in Lob OpenAPI on GitHub, then select "Monitor API contracts" from the left sidebar to geto to [this page](https://github.com/lob/lob-openapi/actions/workflows/monitor.yml). Click on any of the instances shown to see the results of the scheduled tests, which are divided into three sections by goalie (Address Verification, Print and Mail API, and Billing).
```
Comment (hilary) [inline MONITORING.md:15]: Typo "the left sidebar to get to" rather than "to geto to"

File: ALERTS.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
+# Contributing Guide
+
+- [Introduction](#introduction)
+- [Configuration](#configuration)
+- [Slack Web API](#slack-web-api)
+- [Error Message](#error-message)
+
+## Introduction
+
+The reason monitoring tests are separated by goalie is so that, in case of an incident, the person on-call can find the failing test easily. But how can they know when a test is failing unless they wade into the GitHub actions page and manually search for a failing test? That's where alerts come in.
```
Comment (hilary) [inline ALERTS.md:10]: This paragraph just doesn't work for me. Sorry. In keeping with your tone, how about

"If a contract test fails during a monitoring run, the monitor sends an alert to the Slack channel for the relevant on-call goalie. The rest of this guide explains the previous sentence."

File: ALERTS.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Contributing Guide
```
Comment (hilary) [inline ALERTS.md:1]: Oops! (heh)

```
# Alerts
```

File: MONITORING.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
+# Contributing Guide
```
Comment (hilary) [inline MONITORING.md:1]: You really like that title, don't you ;)

```
# Contract Monitoring
```

**Review by hilary — CHANGES_REQUESTED** [review]

File: ALERTS.md (hunk 2 above)
Comment (sudoku-lord) [reply ALERTS.md:10]: That's much more concise, thank you! I wasn't really sure how to phrase the intro, so this helped.

**Review by hilary — APPROVED** [review]
:shipit:

**Pull request merged**